test/compilable/extra-files/makedeps.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+GDEPFILE=${RESULTS_DIR}/compilable/makedeps.dep
+GDEPFILE_a=${RESULTS_DIR}/compilable/makedeps_a.dep
+grep  'makedeps_[0-9]*.o:' ${GDEPFILE} ||
```

Comment:
I've added a comment explaining this case - on Windows (and maybe other platforms) the object file extension is .obj instead of .o, so this just allows to grep over the two cases clearly without resorting to extended regular expressions.